packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -311,15 +311,21 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         </>
       );
       break;
-    case unsupportedEnvironmentPageTrampoline():
-      pageElement = (
-        <>
-          {
-            /* @conditional-compile-remove(unsupported-browser) */
-            <UnsupportedBrowserPage onTroubleshootingClick={props.options?.onEnvironmentInfoTroubleshootingClick} />
-          }
-        </>
-      );
+  }
+
+  /* @conditional-compile-remove(unsupported-browser) */
+  if (
+    adapter.getState().features?.unsupportedEnvironment &&
+    adapter.getState().environmentInfo?.isSupportedBrowser === false
+  ) {
+    pageElement = (
```

Comment:
if this is a page in of itself, should we be using adapter state page to drive this?

Reply:
if so, we should check that `onCallEnded` fires correctly